pkg/lib/cockpit.d.ts

```diff
@@ -30,6 +30,19 @@ declare module 'cockpit' {
 
     export let language: string;
 
+    interface Transport {
+        csrf_token: string;
+        origin: string;
+        host: string;
+        options: JsonObject;
+        uri(suffix: string): string;
+        wait(callback: (transport: Transport) => void): void;
+        close(problem: string): void;
+        application(): string;
```

Comment:
I discovered these typing issues by just opening a file which uses `cockpit.transport` and seeing if anything was still red :)

I suppose you can do the same by running `tsc` with some grep magic.